doc/portfile.7

```diff
@@ -31,39 +31,49 @@
 .Dd February 13, 2007
 .Dt PORTFILE 7 "MacPorts"
 .Os Darwin
+.
+.
```

Comment:
What is the purpose of these periods?

Reply:
A single dot is an empty request for the roff processor and is ignored. As the commit message says this is used throughout the file to visualize the structure in the roff source.

Reply:
It's very unconventional. I've been writing `mdoc` files for a very long time and it's almost never done.

Reply:
Is it?  I was more than 15 years the maintainer of groff, and I've used such structuring all the time.  Actually, it was me together with Ruslan Ermilov who have completely rewritten the mdoc macros to be consistent and readable, fixing many bugs...

Reply:
BTW, I suggest to have a look into the source code of the [`groff_mdoc.7` man page](http://git.savannah.gnu.org/cgit/groff.git/tree/tmac/groff_mdoc.7.man) to see how such files should look like IMHO.

Reply:
Ah, and please show me the incorrect changes of the *formatting* (not contents).  Maybe I've missed something, or there are typos.

Reply:
It's *very* irritating that you obviously didn't have a closer look at the three commits.  The first one only introduces structure; the second one does the mdoc syntax fixes and improvements, and the third one does the 'useless' formatting, i.e., re-formatting to get decent line lengths, together with using two spaces after a full dot.

Guess why I did this split-up into logical units – it wouldn't cross your mind that, say, the third commit could be omitted, right?

For me, adding the empty dots is almost *everything* – suddenly I can quickly find the next `.Sh` without using a forward search...  IMHO it's a weakness of the NetBSD manual linter that it doesn't enforce at least *a bit* structure to otherwise very hard to read man pages.  Similarly, it's a weakness that it doesn't enforce two spaces after a full dot.

Whatever.  Do what you want with my changes.  I no longer care.

Reply:
> It's very irritating that you obviously didn't have a closer look at the three commits

I went to "files changed", as I always do, and started reading. After a page or two of things that did nothing I stopped. If you want the changes considered separately, then you should submit distinct pull requests.

> For me, adding the empty dots is almost everything – suddenly I can quickly find the next .Sh without using a forward search

I've never had such a problem.